docker-compose.yml

```diff
@@ -28,6 +28,8 @@ services:
       - ./backend:/app
       - /app/.venv
     environment:
+      DEV_MODE_SKIP_AUTH: True
```

Comment:
Please don't hardcode it but make it the default instead. Then, we can still test what happens if it is secured.
> DEV_MODE_SKIP_AUTH: ${DEV_MODE_SKIP_AUTH:-True}